app/views/reports/regions/show.html.erb

```diff
@@ -1,5 +1,4 @@
-<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0/dist/Chart.min.js" integrity="sha256-Uv9BNBucvCPipKQ2NS9wYpJmi8DTOEfTA/nH2aoJALw=" crossorigin="anonymous"></script>
-
+<script src="https://cdn.jsdelivr.net/npm/chart.js@3.9.1/dist/chart.min.js" integrity="sha256-+8RZJua0aEWg+QVVKg4LEzEEm/8RFez5Tb4JBNiV5xA=" crossorigin="anonymous"></script>
```

Comment:
This is a hell of a lot of touchpoints to include this script, and is a bit unconventional that we're loading it from a CDN, instead of installing it into the codebase via npm/yarn. Can you create a story to "revisit loading of chartJS to be more Railsy" for a future maintenance task? No need to change the way we do that in this PR.

Reply:
I agree, I already have a [story](https://app.shortcut.com/simpledotorg/story/9325/explore-cdn-import-statements-do-we-need-all-of-these) as part of the epic to address this so that its not missed 👍